cmd/metal-api/main.go

```diff
@@ -789,7 +812,11 @@ func initHeadscale() {
 }
 
 func dumpSwaggerJSON() {
-	cfg := initRestServices(false)
+	audit, err := createAuditingClient(logger.Named("audit"))
+	if err != nil {
+		logger.Fatal(err)
+	}
```

Comment:
Theoretically this should not be required as this function is only a command for outputting the swagger spec, immediately shutting down the program afterwards.

Reply:
Could be solved by replacing the noop-auditing implementation with nil-implementations